blocks/video/video.js

```diff
@@ -25,7 +25,7 @@ function embedYoutube(url, replacePlaceholder, autoplay) {
   }
 
   const temp = document.createElement('div');
-  temp.innerHTML = `<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.25%;">
+  temp.innerHTML = `<div>
       <iframe src="https://www.youtube.com${vid ? `/embed/${vid}?rel=0&v=${vid}${suffix}` : embed}" style="border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;" 
```

Comment:
should we also use a class here instead of inline style, since we're making that change later in the file?